packages/cache/README.md

```diff
@@ -0,0 +1,42 @@
+# `@prairielearn/cache`
+
+Utilities to help connect to and store information in a cache. This package _does not_ load configurations directly. Instead, configs should be passed in when the package is initialized upon loading the application. Then, the package can be used in the throughout the application to interact with the cache.
+
+## Usage
+
+First, you will need to initialize the library with the cache type that you are intending to use and optionally, a redis server URL if that is the cach type being used:
+
+```ts
+import { cacheInit } from '@prairielearn/cache';
```

Comment:
These docs (and the rest in this file) need to be updated to reflect the new API.

Reply:
Updated the docs to reflect the new API in [cc33414](https://github.com/PrairieLearn/PrairieLearn/pull/9360/commits/cc33414906007727a55854776b4e89552b90b0dd).